test/unit/test_analysis_op.c

```diff
@@ -96,8 +97,63 @@ bool test_rz_analysis_op_val() {
 	mu_end;
 }
 
+bool test_rz_core_analysis_bytes() {
+	RzCore *core = rz_core_new();
+	rz_core_set_asm_configs(core, "x86", 64, 0);
+	ut8 buf[128];
+	int len = rz_hex_str2bin("554889e5897dfc", buf);
+	RzPVector *vec = rz_core_analysis_bytes(core, buf, len, 0);
+	mu_assert_notnull(vec, "rz_core_analysis_bytes");
+	mu_assert_eq(rz_pvector_len(vec), 3, "rz_core_analysis_bytes len");
+
+	RzAnalysisBytes *ab = rz_pvector_at(vec, 0);
+	mu_assert_streq(ab->opcode, "push rbp", "rz_core_analysis_bytes opcode");
+
+	ab = rz_pvector_at(vec, 1);
+	mu_assert_streq(ab->opcode, "mov rbp, rsp", "rz_core_analysis_bytes opcode");
+	mu_assert_streq(ab->pseudo, "rbp = rsp", "rz_core_analysis_bytes pseudo");
+
+	ab = rz_pvector_at(vec, 2);
+	mu_assert_streq(ab->opcode, "mov dword [rbp - 4], edi", "rz_core_analysis_bytes opcode");
+	mu_assert_streq(ab->pseudo, "dword [rbp - 4] = edi", "rz_core_analysis_bytes pseudo");
+
+	rz_core_free(core);
+	rz_pvector_free(vec);
+	mu_end;
+}
+
+bool test_rz_core_print_disasm() {
+	RzCore *core = rz_core_new();
+	rz_core_set_asm_configs(core, "x86", 64, 0);
+	ut8 buf[128];
+	int len = rz_hex_str2bin("554889e5897dfc", buf);
+	RzPVector *vec = rz_pvector_new((RzPVectorFree)rz_analysis_disasm_text_free);
+	RzCoreDisasmOptions options = {
+		.vec = vec,
+		.cbytes = 1,
+	};
+	mu_assert_notnull(vec, "rz_core_print_disasm vec not null");
+	rz_core_print_disasm(core, 0, buf, len, len, NULL, &options);
+
+	mu_assert_eq(rz_pvector_len(vec), 3, "rz_core_print_disasm len");
+	RzAnalysisDisasmText *t = rz_pvector_at(vec, 0);
+	mu_assert_eq(t->offset, 0, "rz_core_print_disasm offset");
+
+	t = rz_pvector_at(vec, 1);
+	mu_assert_eq(t->offset, 1, "rz_core_print_disasm offset");
+
+	t = rz_pvector_at(vec, 2);
+	mu_assert_eq(t->offset, 4, "rz_core_print_disasm offset");
```

Comment:
check also the ->text and ->arrow. Also, please add some doxygen doc for the ->arrow field because it is not clear what it represents.